packages/core/content-releases/admin/src/components/tests/ReleaseModal.test.tsx

```diff
@@ -67,4 +68,94 @@ describe('ReleaseModal', () => {
     await user.clear(inputElement);
     expect(dialogSaveButton).toBeDisabled();
   });
+
+  it('should show scheduled fields when selecting schedule release', async () => {
+    const handleCloseMocked = jest.fn();
+    render(
+      <ReleaseModal
+        handleClose={handleCloseMocked}
+        handleSubmit={jest.fn()}
+        initialValues={{ name: 'title', date: null, time: '', timezone: '', scheduledAt: null }}
+        isLoading={false}
+      />
+    );
+    const dialogContainer = screen.getByRole('dialog');
```

Comment:
Isn't the only thing you render a dialog? Why do you need to keep doing `within`? 🤔